documentation/getting-started-cli/source/debugging-with-gdb-lldb.rst

```diff
@@ -41,7 +41,7 @@ Understanding name mangling
 
 Dylan identifiers (method and variable names) do not follow the same
 rules as C; they are case insensitive, can contain extra characters
-such as '<' or '-', and are part of the two-level namespace (module
+such as '<' or ‘-’, and are part of the two-level namespace (module
```

Comment:
intentional?

Reply:
Yes, because Sphinx does its smart quotes wrong otherwise - see
https://stackoverflow.com/questions/59135242/how-to-ensure-correct-smart-quotes-in-sphinx
https://github.com/sphinx-doc/sphinx/issues/6884
https://sourceforge.net/p/docutils/bugs/383/
(I've been on their case about this!)